readme.md

```diff
@@ -2,11 +2,13 @@
  
 ### [Bootstrap-plus @ Github.io Website](https://praegus.github.io/fitnesse-bootstrap-plus-theme/)
```

Comment:
Jullie passen alleen maar formatting aan in de readme? Ik zou op z'n minst een kopje toevoegen met een verwijzing naar bovenstaande readme. Anders gaan mensen het niet vinden.

Reply:
Deze hadden we al op regel 5 staan. we hebben deze nu op regel 3 gezet zodat hij meer opvalt.